accounts/accounts_test.go

```diff
@@ -128,7 +128,7 @@ func TestSignRace(t *testing.T) {
 	if err := am.TimedUnlock(a1.Address, "", 15*time.Millisecond); err != nil {
 		t.Fatalf("could not unlock the test account", err)
 	}
-	end := time.Now().Add(80 * time.Millisecond)
+	end := time.Now().Add(500 * time.Millisecond)
 	for time.Now().Before(end) {
```

Comment:
I switched over to Go 1.5, maybe it has a bit different inlining strategy and caused the `Sign` to be inlined and thus not peemted. Dunno really, but the Sleep fix should also be equally good. (Sorry for the late reply, wasn't available when you asked and since it got merged before I got to reply, I figured it can wait, me not having to type on a mobile with shitty cell data access)